tests/unit/test_tap_kafka.py

```diff
@@ -281,19 +281,20 @@ def test_generate_schema_with_composite_pk(self):
             })
 
     def test_generate_catalog_with_no_pk(self):
-        """table-key-properties should be empty list when no PK defined"""
+        """table-key-properties should not be empty when no custom PK defined"""
```

Comment:
We need to have a test scenario to cover generating catalog when no custom pk nor message keys are available.

Reply:
addressed by [4a35b10](https://github.com/transferwise/pipelinewise-tap-kafka/pull/68/commits/4a35b1085b17c1e8c103e2594b4049271d10ff2e)